packages/discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -3,7 +3,7 @@
 import time
 from datetime import datetime
 from decimal import Decimal
-from typing import List, Optional, Tuple, TypedDict, Union, cast
+from typing import List, Optional, Tuple, TypedDict, cast
```

Comment:
so purchases via user bank are deprecated once and for all? big moment!
noticed the convo about updating min client version, assuming it's for this?

Reply:
whoops just read the PR desc. good to go!